artillery/test.yml

```diff
@@ -0,0 +1,19 @@
+config:
```

Comment:
I think that we need to remove this file since we tested another website - example.com and this check is unnecessary there